taxonomy/utils.py

```diff
@@ -688,3 +695,41 @@ def update_xblock_skills_verification_counts(usage_key: str, verified_skills: Li
         skill_id__in=ignored_skills,
         is_blacklisted=False,
     ).update(ignored_count=F('ignored_count') + 1)
+
+
+def generate_and_store_job_description(job_external_id, job_name):
+    """
+    Generate and store a job description.
+
+    Arguments:
+        job_external_id (str): Lightcast job id
+        job_name (str): Job name
+    """
+    prompt = settings.JOB_DESCRIPTION_PROMPT.format(job_name=job_name)
+    description = chat_completion(prompt)
+    Job.objects.filter(external_id=job_external_id).update(description=description)
```

Comment:
This expression gives an impression that we are saving same description for multiple jobs. Can we use save() instead?

Reply:
I think `update` will be more performant because `filter` and `update` will happen in a single call.